src/types.ts

```diff
@@ -0,0 +1,352 @@
+export interface User {
```

Comment:
There are a few types (`Repo` in BotConfig) scattered elsewhere.  Are they intentionally not part of this file?